templates/articles/article_index_page.html

```diff
@@ -1,13 +1,7 @@
 {% extends 'base_page.html' %}
 
-{% load static %}
-{% load i18n %}
-{% load wagtailcore_tags %}
-{% load wagtailimages_tags %}
-{% load wagtailmetadata_tags %}
-{% block meta_tag %}
-  {% meta_tags %}
```

Comment:
Do we not need to load the `meta_tags` output at all?

Reply:
Never mind - you've done it later on! 🤦🏼‍♂️

https://github.com/nationalarchives/ds-wagtail/pull/1375/files#diff-690b81fad8df2a1f1ce37c846641d9247e3472d244f17039d2910a5ed0c98d5eR23-R25